server/game/gameSystems/LookMoverDeckCardsTopOrBottomSystem.ts

```diff
@@ -0,0 +1,81 @@
+import type { AbilityContext } from '../core/ability/AbilityContext';
+import type { Card } from '../core/card/Card';
+import { CardTargetSystem, type ICardTargetSystemProperties } from '../core/gameSystem/CardTargetSystem';
+import { GameEvent } from '../core/event/GameEvent';
+import { Location } from '../core/Constants';
+import { lookAt, moveCard } from './GameSystemLibrary';
+
+export interface ILookMoveDeckCardsTopOrBottomProperties extends ICardTargetSystemProperties {
+    amount: number;
+}
+
+export class LookMoveDeckCardsTopOrBottomSystem<TContext extends AbilityContext = AbilityContext> extends CardTargetSystem<TContext, ILookMoveDeckCardsTopOrBottomProperties> {
+    public override readonly name = 'LookMoveDeckCardsTopOrBottomSystem';
+
+    protected override defaultProperties: ILookMoveDeckCardsTopOrBottomProperties = {
+        amount: 1,
```

Comment:
Right now the property `amount` is required, so there is no point providing a default because the caller always has to give it a value. We should either change it to `amount?: number` in the properties or remove this default.

The decision depends on whether we think it is useful to make the parameter optional and just do 1 card by default. I would rather be explicit in this case since I don't think there's a default number that is common, but it's up to you.

Reply:
remove the default since we always need to specify how many.